src/electron/preview-renderer.tsx

```diff
@@ -120,8 +119,16 @@ export function render(init: RenderInit): void {
 		public render(): JSX.Element | null {
 			const props = this.props as InjectedPreviewComponentProps;
 			const contents = props.contents || {};
-			const children = typeof contents.default === 'undefined' ? [] : contents.default;
-			const slots = omit(contents, ['default']);
+
+			const renderedContents: { [slotId: string]: React.ReactNode[] } = {};
+			Object.keys(contents).forEach(slotId => {
```

Comment:
Please use `reduce` for simplicity here, also make it clear that `default` lands in `children`

```js
const renderedSlots = Object.keys(omit(contents, ['default'])).reduce((renderedSlots, slotName) => {  
  //  and so forth
});
```